share/revng/test/configuration/revng-qa/compile.yml

```diff
@@ -1,27 +1,29 @@
 tags:
-  - name: clang
+  - name: clang-only
   - name: simple-executable
   - name: stripped
   - name: with-debug-info
 commands:
   - type: revng-qa.compiled-with-debug-info
     from:
       - type: source
-        filter: linux and simple-executable and !clang
+        filter: linux and simple-executable and !clang-only
```

Comment:
If `apple-aarch64` does not have the `linux` tag, this won't be necessary.

Reply:
This is not related to apple though. I just renamed `clang` tag to `clang-only`, because it was confusing to me.